webui/react/src/pages/Admin/UserManagement.tsx

```diff
@@ -464,10 +465,10 @@ const UserManagement: React.FC = () => {
                   width={170}
                   onChange={handleStatusFilterApply}
                 />
-              </Columns>
+              </Row>
             </Column>
-            <Column align="right">
-              <Columns>
+            <Column align="right" width="hug">
```

Comment:
fix: i don't understand how making the container hug the contents is supposed to move the "Actions" and "Add User" buttons to the right
![Screenshot 2023-11-14 at 16-33-08 Determined](https://github.com/determined-ai/determined/assets/15078396/8a95e5ea-4180-4989-a984-ba1a28ede4dc)